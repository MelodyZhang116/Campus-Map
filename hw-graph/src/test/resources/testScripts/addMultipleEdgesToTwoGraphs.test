# test that multiple edges can be added to two graphs and they will not affect each other.

# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2

# Add an edge
AddEdge graph1 n1 n2 e1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1

# Create another graph
CreateGraph graph2

# Add a pair of nodes
AddNode graph2 n3
AddNode graph2 n4

# Add another edge
AddEdge graph2 n3 n4 e2

# Print the nodes in the graph2 and the outgoing edges from n3
ListNodes graph2
ListChildren graph2 n3

# Print the nodes in the graph1 and the outgoing edges from n1, to make sure that
# edge in graph1 is not affected by the edge in graph2
ListNodes graph1
ListChildren graph1 n1