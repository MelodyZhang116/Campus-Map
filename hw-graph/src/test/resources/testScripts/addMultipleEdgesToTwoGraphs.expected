# test that multiple edges can be added to two graphs and they will not affect each other.

# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1

# Add an edge
added edge e1 from n1 to n2 in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1 n2
the children of n1 in graph1 are: n2(e1)

# Create another graph
created graph graph2

# Add a pair of nodes
added node n3 to graph2
added node n4 to graph2

# Add another edge
added edge e2 from n3 to n4 in graph2

# Print the nodes in the graph2 and the outgoing edges from n3
graph2 contains: n3 n4
the children of n3 in graph2 are: n4(e2)

# Print the nodes in the graph1 and the outgoing edges from n1, to make sure that
# edge in graph1 is not affected by the edge in graph2
graph1 contains: n1 n2
the children of n1 in graph1 are: n2(e1)

